.idea/.name

```diff
@@ -0,0 +1 @@
+admin-core
```

Comment:
@mccleaveryan Idea files still coming into this branch...

Reply:
@mccleaveryan you need to clear down the git cache: https://richardjoo.wordpress.com/2012/12/21/how-to-clear-git-cache/
This is happening because the files were committed before the gitignore was added. The repo cache needs reset.